xtask/coverage/src/symbols/msts.rs

```diff
@@ -125,12 +135,12 @@ impl TestCase for SymbolsMicrosoftTestCase {
             debug_text.push_str(" - actual: ");
 
             if let Some(actual) = actual {
-                let name = actual.str(&code).trim();
+                let name = &code[*actual.range()].trim();
```

Comment:
No this change is not needed. It is just to remove code from the crate

Reply:
It doesn't seem we're removing anything. Would it be possible to keep only the changes related to this PR, please? It gets very difficult to review the PR otherwise... for my mental sanity 😢